Ladybird/Qt/BrowserWindow.cpp

```diff
@@ -501,19 +500,19 @@ Tab& BrowserWindow::create_new_tab(Web::HTML::ActivateTab activate_tab)
 
     QObject::connect(&tab->view(), &WebContentView::urls_dropped, this, [this](auto& urls) {
         VERIFY(urls.size());
-        m_current_tab->navigate(urls[0].toString());
+        m_current_tab->navigate(ak_url_from_qurl(urls[0]));
 
         for (qsizetype i = 1; i < urls.size(); ++i)
-            new_tab(urls[i].toString(), Web::HTML::ActivateTab::No);
+            new_tab_from_url(ak_url_from_qurl(urls[i]), Web::HTML::ActivateTab::No);
     });
 
-    tab->view().on_new_tab = [this](auto activate_tab) {
-        auto& tab = new_tab("about:blank", activate_tab);
+    tab->view().on_new_web_view = [this](auto activate_tab, Web::HTML::WebViewHints, Optional<u64>) {
```

Comment:
This also seems to be the wrong commit

Reply:
gah, not sure how I screwed up the fixups this badly.